interactions/api/gateway.py

```diff
@@ -366,7 +365,8 @@ def _dispatch_event(self, event: str, data: dict) -> None:  # sourcery no-metric
                 _name: str = _event_path[0] if len(_event_path) < 3 else "".join(_event_path[:-1])
                 __obj: object = getattr(__import__(path), _name)
 
-                if name in {"_create", "_add"}:
+                # name in {"_create", "_add"} returns False (tested w message_create)
+                if any(_ in name for _ in {"_create", "_update", "_add", "_remove", "_delete"}):
```

Comment:
Not sure why this was changed in this PR. Mind explaining to me why its needed to have the client attribute in delete/remove events?

Reply:
It is not really needed, but I thought some users could have some kind of use for it so I added them